backend/charts/api/templates/deployment.yaml

```diff
@@ -79,6 +79,36 @@ spec:
           args:
             - migrate
             - up
+          env:
+            - name: HOST_IP
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.hostIP
+
+          {{- with .Values.migrations.extraEnvVars }}
```

Comment:
I tried running this but the helm output failed due to indentation I believe. I think it might need to be indented one more section so that it appears underneath the `env`.